packages/edit-site/src/components/editor/index.js

```diff
@@ -210,8 +210,8 @@ function Editor() {
 												baseStyles={
 													settings.__experimentalGlobalStylesBaseStyles
 												}
-												contexts={
-													settings.__experimentalGlobalStylesContexts
+												metadata={
+													settings.__experimentalGlobalStylesMetadata
```

Comment:
Previously we discussed passing this type of metadata from the server to the client to promote reusability. @youknowriad referred he did not like the idea of having metadata (like presets structure or style properties) as a setting. I'm cc @youknowriad in case he has any thoughts on this change, specifically the passing of STYLE_PROPERTY as a setting from the server.

Reply:
In the interest of agility, I've duplicated in the client the style property metadata. The conversation can continue, and if we want to pass down the metadata to the client we can address it in a follow-up PR.

Reply:
I think what's important for me is that the settings of the `BlockEditorProvider` make sense in isolation, in contexts without global styles, without WordPress and someone passing config there to build his own editor should be able to understand and pass the right config depending on what he wants.

So just make sure, that assumption stands :P